blog/_posts/2017-07-04-pivotal-london-intro.md

```diff
@@ -0,0 +1,57 @@
+---
+title: And so it begins!
+class: page blog
+layout: bootstrap
+current: blog
+permalink: blog/2017-07-04-and-so-it-begins
+author: Emma Deacon and Kara de la Marck
+twitter: EmmaMDeacon KaraMarck
```

Comment:
unfortunately we can only have one twitter account linked to the author 😓 
Do you have a team account? Alternatively, we could link to one this time, and to the second one for the next post?